docs/install/install-serving-with-yaml.md

```diff
@@ -0,0 +1,380 @@
+---
+title: "Installing Knative Serving using YAML files"
+linkTitle: "Install Serving using YAML"
+weight: 02
+type: "docs"
+showlandingtoc: "false"
+---
+
+This topic describes how to install Knative Serving.
+You can install the Knative Serving component by applying YAML files using the `kubectl` CLI.
+
+When installing Knative, you can install the Serving component, Eventing component, or both on your cluster.
+For how to install the Eventing component, see [Installing Eventing using YAML files](./install-eventing-with-yaml.md)
+
+## Prerequisites
+
+Before installation, you must meet the prerequisites.
+See [Knative Prerequisites](./prerequisites.md).
+
+
+## Install the Serving component
+
+To install the serving component:
```

Comment:
I'd remove this line - I think a lot of the same comments from above apply for serving too